cypress/e2e/models/administration/assessment_questionnaire/assessment_questionnaire.ts

```diff
@@ -15,14 +31,61 @@ export class AssessmentQuestionnaire {
         });
     }
 
-    public importQuestionnaire() {
+    public static importQuestionnaire(fileName) {
+        AssessmentQuestionnaire.open();
+        clickByText(button, "Import questionnaire");
+        cy.get('input#yamlFile-file-upload-filename', { timeout: 2 * SEC }).attachFile(fileName, {
+            subjectType: "drag-n-drop",
+        });
+        cy.get("form.pf-v5-c-form", { timeout: 5 * SEC })
+        .find("button")
+        .contains("Import")
+        .click();
```

Comment:
Please try if `clickBYText()` will work here instead

Reply:
It doesn't work here.